block_linux.go

```diff
@@ -361,6 +361,10 @@ func (ctx *context) disks() []*Disk {
 			driveType = DRIVE_TYPE_ODD
 			busType = BUS_TYPE_SCSI
 			storageController = STORAGE_CONTROLLER_SCSI
+		} else if strings.HasPrefix(dname, "xvd") {
```

Comment:
@retr0h @mvisonneau if you'd like, please consider adding a code comment here that explains this block is for AWS block device naming and/or link to a URL that explains the AWS naming convention.